engines/hadesch/ambient.cpp

```diff
@@ -0,0 +1,357 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * Copyright 2020 Google
```

Comment:
Congrats on getting to the PR stage. First comment.. double-check your comments. Not sure where the "Copyright 2020 Google" has come from. As far as I'm aware, that shouldn't be part of it.

Reply:
I added this as per my employer recommendation this makes it easier for me to contribute

Reply:
You and/or your employer do not understand how GPLv2 works. If you do work for Google, I suggest you talk to whatever internal team is responsible for licensing compliance -- I am sure they will explain to you how this process works correctly.

Reply:
I concur. It really makes it seem like it's claiming ScummVM belongs to Google. Which is patently false. If you work for Google and worked on this engine during office hours and want to give credit to them, it might be better to put in a separate comment in the engine header file instead

Reply:
You could achieve the same if you do it similar to, let's say https://github.com/scummvm/scummvm/blob/master/engines/sword25/sword25.cpp#L23

Where we have the standard ScummVM GPLv2+ header followed by additional copyright.

Another good example is
https://github.com/scummvm/scummvm/blob/master/common/dct.h#L23

Reply:
Note that the requirement from Google for new files is as follows:

The above copyright notice or a "project authors" style copyright notice to new files, IF permitted by the project maintainers.

Reply:
@phcoder So, what is your resolution on this?